Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG CUDA_VERSION="11.8.0"
+ARG CUDNN_VERSION="8"
+ARG UBUNTU_VERSION="22.04"
+ARG PYTHON_VERSION="3.11.4"
+ARG DOCKER_FROM=nvidia/cuda:$CUDA_VERSION-cudnn$CUDNN_VERSION-devel-ubuntu$UBUNTU_VERSION
+
+
+
+FROM $DOCKER_FROM AS base
+
+RUN apt-get update && apt-get install -y \
+    wget \
+    bzip2 \
+    git \
+    ca-certificates \
+    --no-install-recommends && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /tmp
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
```

Comment:
You can just use the official docker image of miniconda:

COPY --from=continuumio/miniconda3:23.5.2-0 /opt/conda /opt/conda

Reply:
actually handy - thanks